### PR TITLE
Update wf and tags

### PR DESCRIPTION
**Pull request by psbrar99**
- Add a new column for publishing tags
- relist fips build details without download links for artifacts
- update manifest.json path

### CODE REVIEW

**Comment by psbrar99** [comment]
the changes can be seen here:- https://f9f3-2001-56a-7cb1-e100-5888-ff17-3004-9a44.ngrok-free.app/download

image

File: layouts/shortcodes/downloads.html [hunk 1]
```diff
@@ -62,9 +63,10 @@
 
   {{ range $valid_version := $.Scratch.Get "latest_versions" }}
   {{ range $manifest.istio_distributions }}
-  {{ if and (findRE $valid_version .version) (ne $os_type "unspecified") (ne .flavor "tetratefips")}}
+  {{ if and (findRE $valid_version .version) (ne $os_type "unspecified") }}
   {{ $release_url := index (.release_notes) 0}}
   <tr>
+    <td> {{ .version }}-{{ .flavor }}-v{{ .flavor_version }} </td>
```
Comment (AlfredoFausto) [inline layouts/shortcodes/downloads.html:69]: Not sure how it gets flavor_version here? The URLs are broken has a double "0" e.g.  tetrate-v00

Comment (AlfredoFausto) [reply layouts/shortcodes/downloads.html:69]: Disregard, I think I found it.

**Review by PetrMc — APPROVED** [review]
thanks for doing this @psbrar99

question (for my own education)- what is the difference between `"https://istio.tetratelabs.io/getmesh/manifest.json"` and `"https://raw.githubusercontent.com/tetratelabs/getmesh/main/site/manifest.json"`

**Comment by PetrMc** [comment]
also there is missing `Istioctl` - probably removing `Istio Distro` makes the most sense?

image
image

**Comment by psbrar99** [comment]
> also there is missing `Istioctl` - probably removing `Istio Distro` makes the most sense?
> 
> image image

thank you @PetrMc for catching this. fixed now.

**Comment by psbrar99** [comment]
> thanks for doing this @psbrar99
> 
> question (for my own education)- what is the difference between `"https://istio.tetratelabs.io/getmesh/manifest.json"` and `"https://raw.githubusercontent.com/tetratelabs/getmesh/main/site/manifest.json"`

@PetrMc : this resolves https://istio.tetratelabs.io/getmesh/manifest.json to https://raw.githubusercontent.com/tetratelabs/getmesh/main/site/manifest.json". this is configured in netlify.toml

**Pull request merged**